botfront/imports/ui/components/nlu/instances/Instances.jsx

```diff
@@ -23,49 +22,40 @@ class Instances extends React.Component {
         this.state = {
             confirmOpen: false,
             editing: NONE,
+            deleting: NONE,
```

Comment:
As you know, you just have to show the confirm if the the index of the to be deleted instance is the right one